packages/runtime/container-runtime/src/connectionTelemetry.ts

```diff
@@ -375,6 +414,25 @@ class OpPerfTelemetry {
 			this.clientSequenceNumberForLatencyStatistics = undefined;
 			this.latencyStatistics.delete(message.clientSequenceNumber);
 		}
+
+		if (isRuntimeMessage(message)) {
```

Comment:
I think you want to sample all messages. I.e. why runtime messages only? Noop logic does not differentiate between then, so it feels like counting telemetry should not either.

Reply:
This check is also used by no-op heuristics logic, so continuing that pattern here.
 Ref: https://github.com/microsoft/FluidFramework/blob/main/packages/loader/container-loader/src/noopHeuristic.ts#L66